host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -227,6 +227,11 @@ defmodule HostCore.WebAssembly.Imports do
     %{token | authorized: Enum.member?(claims.caps, namespace)}
   end
 
+  # allow actor-to-actor calls
+  defp authorize_call(token = %{target: {:actor, _, _}}) do
+    %{token | authorized: true}
+  end
```

Comment:
So if the target is an actor, claims and namespace are irrelevant. Looks like this is what we had before 👍🏻

Reply:
Yep, at least until we support extensibility with OPA calls